packages/forms/backend/sirius-components-forms/src/main/java/org/eclipse/sirius/components/forms/description/FormDescription.java

```diff
@@ -47,6 +48,8 @@ public final class FormDescription implements IRepresentationDescription {
 
     private List<PageDescription> pageDescriptions;
 
+    private VariableManager variableManagerInitializer;
```

Comment:
The initializer is supposed to be a `Function<VariableManager, VariableManager>`. It takes the variable manager, does anything it want with it and returns it. I want to let specifier do things like add new variables conditionally or even change the initial value conditionally for example. Since they will find the `editingContext` and `self` already in the variable manager, they will have the ability to do so. No need to close the API to only predefined variables.